projects/dev.yaml

```diff
@@ -22,7 +22,7 @@ paths:
                       response_cache_control: '{{options.purged_cache_control}}'
                       cx_host: '{{options.transform.cx_host}}'
                 - path: v1/common_schemas.yaml # Doesn't really matter where to mount it.
-                - path: v1/mobileapps.yaml
+                - path: v1/mobileapps.js
```

Comment:
Since we are still keeping around the old projects, all of them should have the same change as `dev.yaml`